### ISSUE TITLE
error: Failed to migrate database! Cannot find module '../../src/arch/db/migrator'

### ISSUE THREAD

**Issue by MichaelBuhler** [ardriveapp/turbo-upload-service#4]
With `MIGRATE_ON_STARTUP=true`, this error occurs:

```
2024-03-11 14:22:31 error: Failed to migrate database! Cannot find module '../../src/arch/db/migrator'
2024-03-11 14:22:31 Require stack:
2024-03-11 14:22:31 - /usr/src/lib/migrations/20231106203141_nullable_content_type.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/migrations/util/import-file.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/migrations/common/MigrationsLoader.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/migrations/migrate/sources/fs-migrations.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/migrations/migrate/migrator-configuration-merger.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/migrations/migrate/MigrationGenerator.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/migrations/migrate/Migrator.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/knex-builder/make-knex.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/execution/transaction.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/client.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/knex-builder/Knex.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/lib/index.js
2024-03-11 14:22:31 - /usr/src/node_modules/knex/knex.js
2024-03-11 14:22:31 - /usr/src/lib/arch/architecture.js
2024-03-11 14:22:31 - /usr/src/lib/server.js
2024-03-11 14:22:31 - /usr/src/lib/index.js
```

Root cause: https://github.com/ardriveapp/turbo-upload-service/pull/2/files#diff-5b535ebf9a48fd75b8d43b14b29c11481583eef7c46e505b78526fc8ef714a1eR19

**Comment by dtfiedler** [comment]
thanks @MichaelBuhler - we'll roll this into our next release

**Comment by MichaelBuhler** [comment]
Fixed in 395d9144e1f496d4175402adf97091bb553e5fa5
